yascheduler/remote_machine/remote_machine.py

```diff
@@ -218,7 +218,7 @@ async def with_limit(conn: SSHClientConnection, fn: SSHCheck):
             await aall(amap(lambda y: with_limit(conn, y), x.checks)) for x in ADAPTERS
         ]
 
-        for candidate, check in zip(ADAPTERS, checks, strict=True):
+        for candidate, check in zip(ADAPTERS, checks):
```

Comment:
Yes, but ignore flake8 check:
```suggestion
        for candidate, check in zip(ADAPTERS, checks):  # noqa: B905
```